tests/ttnn/unit_tests/operations/ccl/perf/perf_run.py

```diff
@@ -0,0 +1,54 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import subprocess
+import re
+from tabulate import tabulate
+from perf_csv import perf_report
+
+
+def validate_results(df):
+    reference_values = {
+        "HOST DURATION [ns]": 10000,
+        "Cycles Count": 120000,
+        "OP TO OP LATENCY [ns]": 3500,
+        "DEVICE FW DURATION [ns]": 115000,
+        "DEVICE KERNEL DURATION [ns]": 110000,
+    }
+
+    averages = df.iloc[-1]
+
+    # for column, reference in reference_values.items():
```

Comment:
can this be deleted or should it be kept in?

Reply:
Yes this can be removed